sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -826,6 +842,21 @@ private boolean isRollingSessionReceiver() {
         private ServiceBusReceiverClientBuilder() {
         }
 
+        /**
+         * Enables auto-complete and auto-abandon of received messages. By default, a successfully processed message is
+         * {@link ServiceBusReceiverAsyncClient#complete(ServiceBusReceivedMessage) completed}. If an error happens when
+         * the message is processed, it is
+         * {@link ServiceBusReceiverAsyncClient#abandon(ServiceBusReceivedMessage) abandoned}.
+         *
+         * @param enableAutoComplete True to enable auto-complete and false otherwise.
+         *
+         * @return The modified {@link ServiceBusReceiverClientBuilder} object.
+         */
+        public ServiceBusReceiverClientBuilder enableAutoComplete(boolean enableAutoComplete) {
```

Comment:
should we say `disableAutoComplete` because by default it it enable ? And do we need a boolean as args, it is already enabled and user can disable it.  
`disableAutoComplete()` ?